src/state/results/selectors.js

```diff
@@ -1,4 +1,11 @@
-import { values } from 'lodash';
+export const getResults = state => {
+	const results = state.results;
 
-export const getResults = state =>
-	values( state.results ).sort( ( a, b ) => b.id - a.id );
+	if ( ! results ) {
+		return [];
+	}
+
+	return Object.keys( results )
+		.map( key => results[ key ] )
```

Comment:
I was nervous to make that change since I couldn't see a clear browser-list for the project. Also I didn't check, but I thought that `Object.values` wouldn't be transpired which is why I left it in here.

Given those concerns, should we go ahead and update it or wait until we have an explicit browser support listing in the project?

Reply:
If you're concerned about `Object.values`, then you'll need to go full ES5 and avoid arrow functions, `const`, etc. Otherwise, if we're assuming a baseline of ES2015 (aka ES6), then `Object.values` is supported.

Reply:
Also despite trying to fix the auto-correct once already, it looks like I still managed to write "transpired" instead of "transpiled." I'm leaving the typo for historical record 🙃